src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -73,7 +73,12 @@ public enum IndexComponent
     TOKEN_VALUES("TokenValues"),
 
     /**
-     * An on-disk block packed index containing the starting and ending rowIds for each partition.
+     * An on-disk block packed index mapping rowIds to partitionIds.
+     */
+    PARTITION_ROWS("PartitionRows"),
```

Comment:
Partly because I'm worried about this being read "rows per partition", I wonder of something like `ROW_TO_PARTITION` would make sense here.